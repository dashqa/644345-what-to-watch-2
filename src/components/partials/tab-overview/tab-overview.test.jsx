import React from 'react';
import renderer from 'react-test-renderer';
import TabOverview from './tab-overview';

const props = {
  rating: 0,
  scoresCount: 0,
  description: ``,
  director: ``,
  starring: [],
};

it(`Overview tab correctly renders after relaunch`, () => {
  const tree = renderer.create(<TabOverview {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
