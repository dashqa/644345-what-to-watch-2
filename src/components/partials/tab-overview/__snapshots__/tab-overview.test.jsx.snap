// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview tab correctly renders after relaunch 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      0
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Bad
      </span>
      <span
        className="movie-rating__count"
      >
        0
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        
      </strong>
    </p>
  </div>,
]
`;
